update/process/metaBlock.go

```diff
@@ -82,6 +90,7 @@ func (m *metaBlockCreator) CreateNewBlock(
 	blockBody := &block.Body{
 		MiniBlocks: make([]*block.MiniBlock, 0),
 	}
+	sftVersionLength := core.MinInt(core.MaxSoftwareVersionLength, len(version))
```

Comment:
You can add in core package a method like "TrimSoftwareVersion(version) trimmedVersion" and use it everywhere you needed

Reply:
done